frontend/src/components/BasicTextarea/BasicTextarea.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ChangeEvent, FunctionComponent } from 'react';
+import { Form } from 'react-bootstrap';
+import { FormControlElement } from '../../types/bootstrap';
+
+interface BasicTextareaProps {
+  onChange: (options: string) => void;
+  alerts: { [key: string]: string } | undefined;
+}
+
+export const BasicTextarea: FunctionComponent<BasicTextareaProps> = ({ alerts, onChange }) => {
+  const onTextareaChange = (event: ChangeEvent<FormControlElement>) => {
+    onChange(event.currentTarget.value);
+  };
+
+  return (
+    <Form.Group>
+      <Form.Label className="bmd-label-floating">Fill in your query below:</Form.Label>
+      <Form.Control name="description" as="textarea" onChange={onTextareaChange} />
```

Comment:
If you're going to build a generic component, then everything in it must be generic ... `BasicTextarea` makes it sound like I can use this anywhere I want a text area ... not the case since it has a fixed label and name.

Reply:
Look through some of the other feedback before u process this one though ... have a feeling u won't need it.

Reply:
I found `DraftJS` overkill since both `react-bootstrap` and `semantic-ui` had textareas; 

Updated label to be a `prop`